scalr/provider.go

```diff
@@ -191,7 +191,6 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 
 	headers := make(http.Header)
 	headers.Add("User-Agent", providerUaString)
-	headers.Add("Prefer", "profile=internal")
```

Comment:
@penja because we added this line in the epic branch:)
The client is createad from default `go-scalr` config
https://github.com/Scalr/terraform-provider-scalr/blob/6c54449b83be5b312f69f70b5454eb8c1124cdaa/scalr/provider.go#L189
It has `preview` profile header.
https://github.com/Scalr/go-scalr/blob/d3d0e86869b387e6e16c9bdb0b51f07bc95af0bd/scalr.go#L104